source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: "Core Concepts"
+description: "How Home Assistant works."
+---
+
+Home Assistant, like all home automation software, allows you to control smart devices in your home. This page will explain the parts of Home Assistant and how they relate.
+
+## Devices
+
+In Home Assistant, a **device** is a physical object in your connected home. An example would be a lightbulb, a Chromecast, or a smart hub.
+
+Home Assistant has a **discovery** feature, which scans for new devices as they’re added and adds them to your Home Assistant automatically.
+
+**Device trackers** are devices like cell phones that can move around.
+
+## Entities
+
+Because many devices do more than one thing, most things in Home Assistant use  **entities**. An entity is a single function or property of a device, like a single sensor, light, or switch.
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
+
+Some entities are useful for controlling things in your home:
+
+- **Switches** let you check if something is switched on or off?
+- **Sensors** can be a wide range of things that watch for changes in the environment, like motion sensors and temperature sensors
+- **Binary sensors** detect something that is either on or off
+- **[Scripts](https://www.home-assistant.io/docs/scripts/)** are special entities that run custom code when they’re triggered
+
+Other entities are there to respond to Home Assistant:
+
+- **Lights** can turn on and off, and sometimes dim and change color
+- **Covers** are things that can move up and down, like garage doors and smart blinds
```

Comment:
Why are these here? And switch is listed above? Why is scripts (which is considered highly advanced) listed above and a basic common thing like "Lights" here?

Reply:
I noted that scripts are advanced, but I'd like to keep a mention of them here so that people coming across scripts in Forums understand how they fit in. Maybe later there can be an "advanced concepts" page that's for more experienced people, but not developers?

Where should switches go? I just don't know how it works so I'd love a suggestion 😄